python/lib/client/dmod/client/dmod_client.py

```diff
@@ -115,11 +158,12 @@ def _extract_dataset_domain(**kwargs) -> DataDomain:
                 raise ValueError(f"Could not deserialize JSON in 'domain_file' `{domain_file!s}` to domain object")
             else:
                 return domain
-        else:
-            try:
-                return DataDomain(**kwargs)
-            except Exception as e:
-                raise RuntimeError(f"Could not inflate keyword params to object due to {e.__class__.__name__} - {e!s}")
+        try:
+            return DataDomain(**kwargs)
+        except Exception:
+            pass
```

Comment:
Eh, this previously was the last resort:  see if the CLI args were actually a serialized domain (perhaps a superset of one). But now it isn't, so if that fails (e.g., because we want auto detection to be run), we don't want to log the exception. 

On the other hand, we have to try manually first, otherwise we don't have a way to designate that method over detection (unless we add more explicit arg parsing).

Reply:
I see your point. Maybe we don't log at `info`, but this kind of information at a `debug` level would be useful.

Reply:
Fair enough.  I've added something here.